ablySpec/RealtimeClientChannel.swift

```diff
@@ -950,7 +950,7 @@ class RealtimeClientChannel: QuickSpec {
                             expect(messageSent.messages![0].clientId).to(beNil())
 
                             let messageReceived = transport.protocolMessagesReceived.filter({ $0.action == .Message })[0]
-                            expect(messageReceived.messages![0].clientId).to(beNil())
+                            expect(messageReceived.messages![0].clientId).to(equal(options.clientId))
```

Comment:
Yes, this changed recently (https://github.com/ably/docs/pull/91), it should've commented it in the commit.